packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -525,8 +532,10 @@ class BallisticScrollActivity extends ScrollActivity {
   BallisticScrollActivity(
     super.delegate,
     Simulation simulation,
-    TickerProvider vsync,
-  ) {
+    TickerProvider vsync, {
+    double initVelocity = 0.0,
+    double initPosition = 0.0,
```

Comment:
From original PR: These need to be documented

Source: https://github.com/flutter/flutter/pull/100133/files#r844473890